dkist/dataset/tiled_dataset.py

```diff
@@ -124,12 +126,19 @@ def flat(self):
         """
         return type(self)(self._data.flat, self.inventory)
 
+    @property
+    def meta(self):
```

Comment:
What does this get us compared to just having it be a normal attribute? Is it just so it's not writeable?

Reply:
Yep